azerothcore/azerothcore-wotlk
# fix(DB/Creature): Changed the spell Sneak(11013) to Sneak(22766) for Jadefire Shadowstalker

They were using a wrong sneak spell, they should go into stealth when used



## Changes Proposed:
-  Changed the spell Sneak(11013) to Sneak(22766)
-  Changed the spell Slowing poison (14897) to  Slowing poison(7992)
-  Changed the event type of the "on respawn" stealth to " on reset" so they go into stealth when out of combat too.
-  Updated the comments of their SmartAI to know wich skill are they using

## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/7056
- Closes https://github.com/chromiecraft/chromiecraft/issues/1226

## SOURCE:

https://tbc.wowhead.com/npc=7110/jadefire-shadowstalker#abilities

https://youtu.be/BTP42cFGsT8?t=48
https://youtu.be/f78IXO87pVU?t=59

## Tests Performed:

- .go c id 7110 
- .gm on so you can see them while on stealth
- .gm off so you cant see them now!

TEST

![image](https://user-images.githubusercontent.com/87535580/126474973-543d6d80-fc05-4288-b4d7-b84ae6185b22.png)

## How to Test the Changes:


1.  .go c id 7110 
2. .gm on so you can see them while on stealth
3. .gm off so you cant see them now!



## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
